tutorials/examples/cov2d_experiment

```diff
@@ -0,0 +1,52 @@
+"""
```

Comment:
By leaving out the extension, you're taking the responsibility of making the script executable, but providing the correct shebang line at the top. Make sure to do a chmod before committing, and don't leave it to the user to decipher how to run your example.

Reply:
The reason we're not using the `.py` extension is to prevent the scripts from being run by the test suite right now (since it relies on local data being available). It will be restored once we have the ability to download data in the code.

That being said, I think it would be better to have an explicit extension like `.py.dontrun` here.

Reply:
@janden ... sure, I just meant if the extension is left out for whatever reason, then a shebang line must be included at the top (+ a +x chmod on the file as far as git is concerned).

Reply:
Right. My point is this is a temporary workaround that will disappear in a few weeks (hopefully).

Reply:
Done in new commit 468e7ce.